substrate/keystore/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 
 [dependencies]
-ethcore-crypto = { git = "https://github.com/paritytech/parity.git", default_features = false }
+parity-crypto = { git = "https://github.com/paritytech/parity-common.git", default_features = false }
```

Comment:
You missed renaming an import, [see travis test](https://travis-ci.org/paritytech/polkadot/jobs/402680325#L1050).

Reply:
fixed, thanks!